File: MyWami/src/com/MyWami/util/GetUserId.java
package com.MyWami.util;

import android.content.Context;
import com.MyWami.model.UserModel;
import com.MyWami.sqlite.SQLiteHelper;

/**
 * Created by robertlanter on 5/21/14.
 */
public class GetUserId {

	static public int getUserId (Context context) {
		UserModel userModel;
		SQLiteHelper sqLiteHelper = new SQLiteHelper(context);
		userModel = sqLiteHelper.getUserInfo();

		return userModel.getUserId();
	}

}

File: MyWamiIOS/MyWami/MyWami/GetUserId.swift
//
//  GetUserId.swift
//  MyWami
//
//  Created by Robert Lanter on 4/21/15.
//  Copyright (c) 2015 Robert Lanter. All rights reserved.
//

import Foundation

public class GetUserId {
    
    func getUserId() -> Int {
        let sqliteHelper = SQLiteHelper()
        var userModel = UserModel()

        userModel = sqliteHelper.getUserInfo()!
        return userModel.getUserId()
    }
}